src/components/Menu.js

```diff
@@ -1,27 +1,29 @@
 import { useState } from 'react';
-import { PrimaryNav, Header, Title, } from '@trussworks/react-uswds'
+import { PrimaryNav, Header, Title, NavMenuButton, GovBanner } from '@trussworks/react-uswds'
 import '@trussworks/react-uswds/lib/index.css'
 
-function Header() {
+import styles from './Menu.modules.scss';
+import { NAVIGATION } from '../lib/constants'
+
+function Menu() {
   const [expanded, setExpanded] = useState(false);
   const onClick = () => {
     setExpanded(prvExpanded => !prvExpanded);
   }
-  const menuItems = [
-    <a href="#two" key="two" className="usa-nav__link">
-      <span>Parent link</span>
-    </a>,
-    <a href="#three" key="three" className="usa-nav__link">
-      <span>Parent link</span>
-    </a>,
-  ];
+  const menuItems = NAVIGATION.map(({name, route}) => {
+    return (
```

Comment:
Not incredibly important but maybe indent the components inside of the return just to fix the spacing and make it slightly more readable.